docs/guides/ibm-circuit-function.ipynb

```diff
@@ -250,6 +250,46 @@
     "}"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "997c3779",
+   "metadata": {},
+   "source": [
+    "## Outputs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "75d7cbba",
+   "metadata": {},
+   "source": [
+    "The output of a Circuit function is a [PrimitiveResult](/api/qiskit/qiskit.primitives.PrimitiveResult), which contains two fields:\n",
+    "\n",
+    "- One or more [PubResult](/api/qiskit/qiskit.primitives.PubResult) objects. These can be indexed directly from the `PrimitiveResult`.\n",
+    "\n",
+    "- Job-level metadata.\n",
+    "\n",
+    "Each `PubResult` contains a `data` and a `metadata` field. \n",
+    "\n",
+    "- The `data` field contains at least an array of expectation values (`PubResult.data.evs`) and an array of standard errors (`PubResult.data.stds`). It can also contain more data depending on the options used.\n",
+    "\n",
+    "- The `metadata` field contains PUB-level metadata (`PubResult.metadata`).\n",
+    "\n",
+    "The below code snippet describes the `PrimitiveResult` (and associated `PubResult`) format."
```

Comment:
```suggestion
    "- The `data` field contains at least an array of expectation values (`PubResult.data.evs`) and an array of standard errors (`PubResult.data.stds`). It can also contain more data, depending on the options used.\n",
    "\n",
    "- The `metadata` field contains PUB-level metadata (`PubResult.metadata`).\n",
    "\n",
    "The following code snippet describes the `PrimitiveResult` (and associated `PubResult`) format."
```